pkg/agent/cniserver/ipam/ipam_delegator.go

```diff
@@ -80,7 +81,19 @@ var defaultExec = &invoke.DefaultExec{
 }
 
 func delegateCommon(delegatePlugin string, exec invoke.Exec, cniPath string) (string, invoke.Exec, error) {
-	paths := filepath.SplitList(cniPath)
+	// The CNI searching paths passed from kubelet.
+	configuredPaths := filepath.SplitList(cniPath)
+	paths := make([]string, len(configuredPaths)+1)
+	// When Antrea agent runs as a Pod, the IPAM plugin is always installed in
+	// DEFAULT_CNI_PATH, but kubelet can be configured to use different paths to
+	// search for CNI plugins. So here we always add DEFAULT_CNI_PATH to the CNI
+	// plugin searching paths to make sure the IPAM plugin installed in the agent
+	// Pod can be found.
+	paths[0] = DEFAULT_CNI_PATH
+	for i, p := range configuredPaths {
```

Comment:
`copy(paths[1:], configuredPaths)`?

Reply:
Thanks for the information. Found append(paths, configuredPaths...) also works.

Reply:
yes, you would just need to change your `make` call to have size 1 and capacity `len(configuredPaths)+1`.